### PR TITLE
update: FIX update change refs variable.

### PR DESCRIPTION
**Pull request by ldjebran**
The current problem is that when a dist 8.5 and we update to 8.6 the change refs is True and the playbook will fail as the ref has not changed.

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by AaronH88 — APPROVED** [review]
/lgtm

**Review by de1987 — APPROVED** [review]

**Comment by omaciel** [comment]
Please add unittests for this before merging

**Comment by omaciel** [comment]
Added donotmerge label

**Comment by omaciel** [comment]
Removed donotmerge label

File: pkg/services/updates.go [hunk 1]
```diff
@@ -785,7 +785,7 @@ func (s *UpdateService) BuildUpdateTransactions(devicesUpdate *models.DevicesUpd
 					if uError != nil {
 						s.log.WithField("error", err.Error()).Error("Error returning current image ostree checksum")
 					}
-					if currentImage.Distribution != updatedImage.Distribution {
+					if config.DistributionsRefs[currentImage.Distribution] != config.DistributionsRefs[updatedImage.Distribution] {
```
Comment (loadtheaccumulator) [inline pkg/services/updates.go:788]: In looking at the tests post-merge, I think this should be a func changesRefs(current string, updated string) bool {} and then simple functional in/out tests for each scenario
- minor to minor (same version e.g., 8.4 to 8.4)
- minor to minor (different version e.g., 8.4 to 8.5)
- minor to major (e.g., 8.5 to 9.0)

Mocking the service(s) becomes unnecessary for this simple test.

**Review by loadtheaccumulator — COMMENTED** [review]
Leaving a note here for future reference and will bring it up separately.

File: pkg/services/updates.go (hunk 1 above)
Comment (loadtheaccumulator) [reply pkg/services/updates.go:788]: @omaciel @ldjebran @AaronH88 catching this post merge, so maybe can be handled in a refactor or just for future reference.

**Pull request merged**